src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function adder(a) {
+    const tmp = sum;
+
+    if (typeof a === 'undefined') {
```

Comment:
```suggestion
    if (a === undefined) {
```
you can do this check even without typeof